src/app/components/fileattachments/index.html

```diff
@@ -1,25 +1,213 @@
-<sky-demo-page
+<sky-tabbed-demo-page
   pageTitle="File Attachment"
   navTitle="File attachment"
-  skyuxModule="SkyFileAttachmentsModule"
-  packageName="@skyux/forms"
 >
 
   <sky-demo-page-summary>
     <p>
       The file attachment module contains two components to add files to forms and display summary information about attachments.
     </p>
+  </sky-demo-page-summary>
+
+  <div class="sky-margin-stacked-content-section">
+	<sky-demo-page-example>
+    <sky-file-attachment-demo></sky-file-attachment-demo>
+  </sky-demo-page-example>
+	</div>
+  
+  <sky-tabset id="content-tabs">
+	
+	<sky-tab
+		tabHeading="Design - single file attachment">	
+	<sky-demo-page-content sectionHeading="Usage">
+	<h3 class="sky-text-success">Use when</h3>
+		<sky-docs-visual-example>
+			<sky-docs-visual-example-text>
+					<p>Use single file attachment when the user needs to upload one file to which they have local access.</p>
+			</sky-docs-visual-example-text>
+			<sky-docs-visual-example-image>
+				<div>
+					<img src="~/assets/img/guidelines/single-file-attach/use.png" />
+					<div class="caption caption-success sky-text-success">Do use single file attachment to upload one file.</div>
+				</div>
+			</sky-docs-visual-example-image>
+		</sky-docs-visual-example>
+		
+	<h3 class="sky-text-danger">Don't use when</h3>
+    <p class="sky-text-block">
+      Don't use single file attachment if the user might need to link to a URL instead of uploading a local file. Use file attachment instead.
+    </p>
+		<sky-docs-visual-example>
+			<sky-docs-visual-example-text>
+					<p>Don't use single file attachment when the user might need to upload more than one file. Use file attachment instead.</p>
+			</sky-docs-visual-example-text>
+			<sky-docs-visual-example-image>
+				<div>
+					<img src="~/assets/img/guidelines/single-file-attach/dont-use-multi.png" />
+					<div class="caption caption-danger sky-text-danger">Don't use single file attachment if the user might upload multiple files.</div>
+        </div>
+			</sky-docs-visual-example-image>
+		</sky-docs-visual-example>
+</sky-demo-page-content>
+  
+  <sky-demo-page-content sectionHeading="Anatomy">
+		<sky-docs-anatomy imagePath="~/assets/img/guidelines/single-file-attach/anatomy.png">
```

Comment:
Should the button in the second half of the image with the Replace file button also have a green 1 on it? I think so unless we consider that a totally different button, and if that's the case, then I think it should have a separate entry in the anatomy list.

Reply:
yup, added